GovUk.Frontend.Umbraco.ExampleApp/uSync/v9/Content/textarea.config

```diff
@@ -6,15 +6,262 @@
     <Trashed>false</Trashed>
     <ContentType>textarea</ContentType>
```

Comment:
I don't think we have any need for blueprints in the example app? Happy to hear your thinking, or otherwise these three files should be removed.